doc/nrf/app_dev/device_guides/nrf54l/vpr_flpr.rst

```diff
@@ -0,0 +1,29 @@
+.. _vpr_flpr_nrf54l:
+
+Working with the FLPR core
+##########################
+
+The nRF54L15 SoC has a dedicated RISC-V CPU (VPR), a fast lightweight peripheral processor (FLPR) designed for software-defined peripherals (SDP).
+This core currently supports the following peripherals:
+
+* GPIO
+* GPIOTE
+* GRTC
+* TWIM
+* UARTE
+* VPR
+
+Running the FLPR core
+*********************
+
+VPR is optimized for implementing SDP.
+The software-defined peripheral running on VPR does not start on its own, but must be started by the application core processor.
+
+.. To be outlined how
+
+Memory allocation
+*****************
+
+If a SDP is active on a VPR CPU, it can lead to increased latency when accessing ``RAM_01``.
+Because of this, when SDP is enabled in a project, you should use ``RAM_01`` to store only the VPR code, data, and other non-time-sensitive information.
```

Comment:
```suggestion
Because of this, when FLPR is used in a project, you should use ``RAM_01`` to store only the VPR code, data, and other non-time-sensitive information.
```